WRQ-31515: Fixed `Scroller`, `Slider`, `VirtualList` and `VirtualGridList` to set prop value to default when `undefined` is passed for the prop value (#1658)
enactjs/sandstone: CHANGELOG.md

The following is a curated list of changes in the Enact sandstone module, newest changes on the top.

## [unreleased]

### Fixed

> - `sandstone/Scroller`, `sandstone/Slider`, `sandstone/VirtualList` and `sandstone/VirtualList.VirtualGridList` to set prop value to default when `undefined` is passed for the prop value

Reviewer: ```suggestion
- `sandstone/Scroller`, `sandstone/Slider`, `sandstone/VirtualList` to have default prop when `undefined` prop is passed
```

Reviewer: We can refer `VirtualList` and VirtualGridList` to `sandstone/VirtualList`. :)

Author: Thank you. I fixed to use sandstone/VirtualList
